Engine/Entities/Ants/Ant.cs

```diff
@@ -154,6 +154,16 @@ public void EmitFoodPheromone()
             Transform foodTransform = new(Transform.Position, 0, Vector2.One);
             World.AddEntity(new FoodPheromone(Name, foodTransform, World, PheromoneTimeSpan));
         }
+
+        private void CheckEntitiesInRegion<T>(int x, int y, IList list)
+        {
+            foreach (Entity e in World.Regions[x][y])
+            {
+                if (e is not T) continue;
+                if (!(e.Transform.GetDistance(Transform) <= PerceptionDistance)) continue;
```

Comment:
We do not care about perception distance when we get entities in a region

Reply:
This is an extracted method that allows us to get the entities in a specific region that are also in range of the ant.